opal/mca/event/libevent2021/libevent2021_component.c

```diff
@@ -120,12 +120,14 @@ const opal_event_component_t mca_event_libevent2021_component = {
     }
 };
 
+#define BUFF_SIZE 1024
```

Comment:
@jsharpe How about using BUFSIZ, which is defined in `stdio.h`?  (it just avoids one more #define)